options.html

```diff
@@ -136,6 +136,37 @@
 {
   box-shadow: 0px 0px 10px #e03030;
 }
+#userFilterLists {
+    margin-bottom: 8px;
+}
+
+#whitelistForm #newWhitelistDomain,
+#customFilterForm #newFilter,
+#subscriptionSelector,
+#customSubscriptionTitle,
+#customSubscriptionLocation
+#excludedDomainsBox,
+#userFiltersBox,
+#rawFiltersTextWrapper {
+    width: 500px;
+    box-sizing: border-box;
+}
+#rawFiltersButtonWrapper {
+    font-size: 90%;
+    text-align: right;
+}
+
+#excludedDomainsBox,
+#userFiltersBox {
+    width: 500px;
```

Comment:
Specifying `width` and `box-sizing` is not necessary here because you already applied those styles above.